scripts/update-correios-wsdl

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+
+import argparse
+
+from correios.update_wsdl import update_wsdl, WSDL_PATH
+
+parser = argparse.ArgumentParser(description='Updates Correios WSDL files')
+parser.add_argument(
+    '-p',
+    '--path',
+    dest='wsdl_path',
+    default=WSDL_PATH,
+    help='Custom path where wsdl files will be saved, '
+        'note that this option will have higher priority than the value of '
+        'the envvar \'CORREIOS_WSDL_PATH\''
+)
+
+args = parser.parse_args()
+
+print('Updating Correios WSDL')
+print(f'Files will be saved on: {args.wsdl_path}')
```

Comment:
IMO we should use only 1 way to format strings and we're already doing that with `str.format`. Can you change it here and in the line below?